samples/sensor/mcp9600/README.rst

```diff
@@ -0,0 +1,49 @@
+.. _mcp9600:
+
+mcp9600 Temperature Sensor
+###################################
+
+Overview
+********
+
+This sample application periodically reads temperature data from the first
+available device that implements SENSOR_CHAN_AMBIENT_TEMP.
+
+Building and Running
+********************
+
+This sample application uses an mcp9600 sensor connected to a board via I2C.
+We used the Sparkfun `MCP9600 Sensor`_ breakout board. The datasheet is
+available at `MCP9600 Datasheet`_
+
+.. _`MCP9600 Sensor`: https://www.sparkfun.com/products/16295
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/sensor/mcp9600
+   :board: frdm_kw41z
+   :goals: flash
+   :compact:
+
+Sample Output
+=============
+To check output of this sample , any serial console program can be used.
+This example uses screen on OS X with a frdm_kw41z development board.
+
+.. code-block:: console
+
+        $ screen /dev/tty.usbmodem0006210000001 115200
```

Comment:
That seems unlikely to be what anybody else would use.  Maybe `/dev/ttyACM0` like most other examples that suggest using screen?

Reply:
Not sure it makes that much of a difference? I do think I say I tested it on a Mac and that's what you get.

Reply:
added additional text.